src/utils/DockerUtil.js

```diff
@@ -126,7 +126,7 @@ export default {
         return;
       }
 
-      containerData.Cmd = image.Config.Cmd || 'bash';
+      containerData.Cmd = image.Config.Cmd || image.Config.Entrypoint || 'bash';
```

Comment:
Unfortunately I don't think this is correct - Doesn't Passing the entrypoint as the cmd to run result in the final cmd be:
`entrypoint + entrypoint`
Previously it was:
`entrypoint + bash`
When no cmd is found, but an entrypoint is, It should just be
`entrypoint`